content/blog/noetic-eol/index.md

```diff
@@ -0,0 +1,178 @@
+
+<img src="/imgs/distro-download-data.png" alt="ROS Downloads by Distro, September 2024" style="width:600px;align:center"/>
+
+### What Does End-of-Life Mean?
+
+End of life for ROS Noetic means that the ROS team will no longer support ROS 1. 
+After May 31st of 2025, the ROS team will no longer provide the following for ROS 1:
+
+
+* New features or capabilities
+* Security updates
+* Bug fixes, patches or support
+* Updated binaries
+
+**After May 31st, 2025 ROS 1 Noetic users may find themselves exposed to un-patched security vulnerabilities and bugs, with few avenues of support short of fixing these issues themselves and recompiling Noetic from source.**
+However, this isn't an outright emergency, it is good to keep mind what won't change after May of 2025.
+
+ROS 1 going end of life does **not** mean:
+
+* *ROS 1 binaries will suddenly disappear.*
+  * ROS 1 binaries will continue to be hosted on `packages.ros.org`
+* *ROS 1 projects will suddenly stop working on May 31st.*
+  * ROS 1 will continue to work, as it does today.
+* *Users must stop using ROS 1 immediately*,
+  * ROS 1 will continue to work, but you should have migration plan as soon as possible.
+
+
+<br>
+
+### ROS 2 Migration Options
+
+**If you are still using ROS 1 it is time to consider switching to ROS 2 Humble Hawksbill ([migration guide](https://docs.ros.org/en/humble/How-To-Guides/Migrating-from-ROS1.html)) or ROS 2 Jazzy Jalisco ([migration guide](https://docs.ros.org/en/jazzy/How-To-Guides/Migrating-from-ROS1.html)).**
+Which distro you choose as your migration target is going to depend on the complexity of your ROS 1 system.
+This [migration heuristic tool](https://github.com/osrf/ros2-porting-heuristics) can help you estimating the complexity of migrating your project to ROS 2.
+If you have a smaller ROS 1 project, or a reasonably de-coupled workspace, it is most likely that you can migrate to ROS 2 Jazzy Jalisco directly.
+The ROS community has produced a number of migration tools which you may find helpful, including this migration tool from [AWS Labs](https://github.com/awslabs/ros2-migration-tools), and this migration tool from ROS maintainer [David Lu!](https://github.com/DLu/roscompile/tree/main/magical_ros2_conversion_tool)
+
+If your ROS 1 system is tightly coupled, has a number of un-ported dependencies, or is reasonably complex, you may want to take advantage of the [ROS 1 to ROS 2 Bridge](https://github.com/ros2/ros1_bridge).
+The ROS 1 to ROS 2 Bridge performs automated translation between major ROS versions and should allow you to migrate your project one package at a time. 
+Due to the operating system constraints for the current active ROS releases (Noetic / Ubuntu  20.04, Humble / 22.04)  the easiest way to use the ROS Bridge is to use ROS 2 Humble Hawksbill compiled from source on Ubuntu 20.04, [which has limited Tier 3 support](https://www.ros.org/reps/rep-2000.html#humble-hawksbill-may-2022-may-2027).
```

Comment:
Looks like my suggestion https://github.com/ros-infrastructure/www.ros.org/pull/54#discussion_r1919895964 was not yet considered.
```suggestion
Due to the operating system constraints for the current active ROS releases (Noetic = Ubuntu  20.04, ROS2 >= Ubuntu 22.04) we suggest [installing ROS-O](https://ros.packages.techfak.net/) next to ROS2 on a decent Ubuntu distro (instead of building ROS2 from source on Ubuntu 20.04).
```
As argued in https://github.com/ros-infrastructure/www.ros.org/pull/54#discussion_r1927672056, I don't think that the easiest way is to build Humble from source. Using ROS-O binary packages is easier (and definitely faster) than compiling everything from scratch.

Reply:
We talked this over internally and we think the best approach here is to present both options to the user. The real problem for users is that we stopped supporting the bridge after Foxy, so the user is already venturing into questionable territory.

Reply:
For me, looks like you stopped support for the bridge too early.